crates/net/network/src/lib.rs

```diff
@@ -12,9 +13,44 @@
 //! In order for a node to join the ethereum p2p network it needs to know what nodes are already
 //! port of that network. This includes public identities (public key) and addresses (where to reach
 //! them).
+//!
+//! ## Usage
+//!
+//! ### Configure and launch the network
```

Comment:
Beautiful. And we'll add docs later on how to use the network, how to instantiate the block provider etc